fireblocks_defi_sdk_py/web3_bridge.py

```diff
@@ -77,11 +77,16 @@ def send_transaction(self, transaction: dict, note="") -> dict:
                 address = self.external_wallet_address
             destination = DestinationTransferPeerPath(ONE_TIME_ADDRESS,
                                                       one_time_address={"address": address})
+        amount = transaction.get("value")
+        if amount:
+            amount = str(Web3.fromWei(amount, 'ether'))
+        else:
+            amount = "0"
```

Comment:
```suggestion
        amount = Web3.fromWei(amount or "0", "ether")
```

Reply:
@rdvorkin are we sure the value in transaction is always in Wei? 
As we are always dealing with contract calls, I think the amount here can always be just set to 0, and we are not really parsing the amount one way or the other (it resides within the transaction data, and we need to decode it to actually find it)